### ISSUE TITLE
Clicking on Screen Reader Optimized status bar has no effect

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#196370]
I have "Screen reader optimised" entry in my status bar.
Clicking on it produces no effect 🐛 

Before it would open a notification and ask me if I am really using a screen reader.
I still think that clicking on it should have some effect. We can be creative what we want to do here.

**Comment by meganrogge** [comment]
Maybe clicking should toggle it?

**Comment by meganrogge** [comment]
I just reverted my change instead
